store/sqlstore/thread_store.go

```diff
@@ -182,7 +182,20 @@ func (s *SqlThreadStore) GetThreadsForUser(userId, teamId string, opts model.Get
 		if opts.Since > 0 {
 			newFetchConditions = sq.And{newFetchConditions, sq.GtOrEq{"Threads.LastReplyAt": opts.Since}}
```

Comment:
(Same as other comment) Should this factor in `ThreadMemberships.LastEditAt`?

Reply:
@calebroseland - there's no LastEditAt in thread membership, so not sure what you meant :)

Reply:
@reflog, doh! I meant `lastupdated` (on `threadmemberships` in DB).  `since` in my interpretation would include changes to `following` (and similar) for sync'ing cached UserThreads in the mobile app.

Reply:
yep, for mobile it would be good to have all changes since `since`, if possible.